src/Creator.tsx

```diff
@@ -379,7 +329,7 @@ export const Creator = ({ NFToupon_Key }: Props) => {
             <Card shadow={false} css={{ borderRadius: '0' }}>
               <Card.Image
                 showSkeleton
-                src={isEmpty(details.imageUrl) ? imageURL : details.imageUrl}
```

Comment:
even this one... i think all the creator code has been deleted... could you please rebase again... and the accept the current change